DevVehicle35-Motors/Interface/IVehicleBased.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace DevVehicle35_Motors
 {
-    internal interface iMainVehicle
+    internal interface IVehicleBased
     {
         public decimal price { get; set; }
```

Comment:
Properties naming convention suggest that they should start with UpperCase.

Reply:
Ok ty, I changed this line